src/pages/index.js

```diff
@@ -3,15 +3,24 @@ import Link from "next/link";
 export default function Home() {
   return (
     <div>
-      <main>
-        <h1>Accessibility Cheatsheets for Components
-        </h1>
+      <main className="home">
+        <div className="home-top">
+          <h1>Accessibility Cheatsheets for Components
+          </h1>
+          <p className="home-top__top-description">
```

Comment:
I am not sure BEM is within the scope of this card. I reviewed it and it did say "semantic HTML" but no BEM was mentioned. I would double check this on apprentice channel or ask @dustin-jw to be sure.